CONTRIBUTING.md

```diff
@@ -17,7 +17,7 @@ If you figure out how to fix the issue then please push the fix to the pull requ
 The [`index.json` file](https://github.com/octokit/webhooks/blob/master/index.json) is generated, please do not edit it. Instead, make changes in the [`payload-examples/api.github.com/` folder](https://github.com/octokit/webhooks/tree/master/payload-examples/api.github.com), then update `index.json` by running the following command
 
 ```
-node bin/octokit-webhooks.js update --cached
+ts-node -T bin/octokit-webhooks.ts update --cached
```

Comment:
Folks won't have the `ts-node` binary installed globally and might not be familiar with it. I'd run the command with `npx` which will use the installed version from our `devDependencies`

```suggestion
npx ts-node -T bin/octokit-webhooks.ts update --cached
```

Reply:
alternatively, we could add an npm script?

Reply:
There actually was one introduced in #186.
```suggestion
npm run build:webhooks
```

Reply:
looks good to me 👍🏼

Reply:
Have updated it to be `npm run build:webhooks -- --cached` :)